app/(home)/components/MakesDataGrid.tsx

```diff
@@ -14,10 +15,35 @@ const columns: GridColDef[] = [
   { field: "modelsCount", headerName: "# of Models", width: 300 },
 ];
 
+const findMaxNum = (x) => {
+  const mymax = 0;
+  for (let i = 0; i < x.length - 1; i++) {
+    if (x[i] > mymax) {
+      mymax = arr[i];
+    } else {
+      continue;
+    }
```

Comment:
we can just remove the `else` block here
```suggestion
    }
```